### DISCUSSION TITLE
Blog posts

### DISCUSSION THREAD

**Discussion by HaydarJawad** [alshedivat/al-folio#2440] [Q&A] [answered]
Hi all, I am having two small issues, I wonder if anyone can help! 

 I have deleted all blogs.md files, and I am using only one template, but I keep getting two things:

1- My only post gets duplicated underneath it, both working.
2- I get this post below, but I don't know where to update it as I cannot find it anywhere in the files

[Displaying External Posts on Your al-folio Blog]
(https://medium.com/@al-folio/displaying-external-posts-on-your-al-folio-blog-b60a1d241a0a?source=rss-17feae71c3c4------2) 
1 min read   ·   April 23, 2022   ·   medium.com

Thank you

**Comment by george-gca** [comment, marked as answer]
1. By duplicated you mean it appears also as featured? If so, just set `featured: false` in the post front matter
2. Comment the references to `external_sources` in [_config.yml](https://github.com/alshedivat/al-folio/blob/8a6ad2d5edbc2604238f833362692f264cca8f0f/_config.yml#L168)

**Comment by HaydarJawad** [comment]
Thank you George, that works great . However, for some reason my next post
with a new name is not appearing .
2018-12-22-distill.md - show
2018-12-23-distill.md - do not show
Regards
Haydar

**Comment by george-gca** [comment]
Note that it takes some time to build and publish the new version of the website. You can check for the status on the `Actions` tab of your repo. Also note that it is good practice to name the file with the post title, so it is easier to identify it.

**Comment by HaydarJawad** [comment]
yes you are absolutely right, it appeared now. I love this package
thank you so much mate! all the best
